lib/spree_signifyd.rb

```diff
@@ -20,7 +20,7 @@ def set_score(order:, score:)
 
   def approve(order:)
     order.contents.approve(name: self.name)
-    order.shipments.each { |shipment| shipment.ready! unless shipment.ready? }
+    order.shipments.each(&:ready!)
     order.updater.update_shipment_state
     order.save!
   end
```

Comment:
This is going deeper than I originally intended, but I'm in agreement. Notice the call site of this approve method (https://github.com/solidusio/solidus_signifyd/blob/master/app/controllers/spree/api/spree_signifyd/orders_controller.rb#L12-L16) there's a real asymmetry. The elseif clause could be `@order.approve!`. I also just noticed that there's no catch-all else clause in this. If Signifyd ever sent a new reviewDisposition, we'd silently ignore it.